api/src/main/java/marquez/db/OpenLineageDao.java

```diff
@@ -135,223 +149,226 @@ default UpdateLineageRow updateMarquezModel(LineageEvent event, ObjectMapper map
     return updateLineageRow;
   }
 
-  default UpdateLineageRow updateBaseMarquezModel(LineageEvent event, ObjectMapper mapper) {
-    NamespaceDao namespaceDao = createNamespaceDao();
-    DatasetSymlinkDao datasetSymlinkDao = createDatasetSymlinkDao();
-    DatasetDao datasetDao = createDatasetDao();
-    SourceDao sourceDao = createSourceDao();
-    JobDao jobDao = createJobDao();
-    JobFacetsDao jobFacetsDao = createJobFacetsDao();
-    DatasetVersionDao datasetVersionDao = createDatasetVersionDao();
-    DatasetFieldDao datasetFieldDao = createDatasetFieldDao();
-    DatasetFacetsDao datasetFacetsDao = createDatasetFacetsDao();
-    RunDao runDao = createRunDao();
-    RunArgsDao runArgsDao = createRunArgsDao();
-    RunStateDao runStateDao = createRunStateDao();
-    ColumnLineageDao columnLineageDao = createColumnLineageDao();
-    RunFacetsDao runFacetsDao = createRunFacetsDao();
+  default UpdateLineageRow updateMarquezModel(DatasetEvent event, ObjectMapper mapper) {
+    daos.initBaseDao(this);
```

Comment:
This is pretty cool! I agree a wrapper class like `ModelDaos` helps with readability and minimizes `DAO` boilerplate initialization code. We can probably take the refactoring further an ask ourselves if we really need all of this logic in the DAO layer? Anyways, great to see cleanup efforts in this part of the codebase! 💯